src/app/admin/theme-image/theme-image.component.ts

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2020-2023 University of Oxford and NHS England
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { MdmResourcesService } from '@mdm/modules/resources';
+import { SecurityHandlerService } from '@mdm/services/handlers/security-handler.service';
+import { MessageHandlerService } from '@mdm/services/utility/message-handler.service';
+import { DomSanitizer } from '@angular/platform-browser';
+import { environment } from '@env/environment';
+import { ApiPropertyResponse, Uuid } from '@maurodatamapper/mdm-resources';
+import { catchError } from 'rxjs';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+export enum ImageChangeType {
+  uploaded,
+  removed,
+  nochange
+}
+
+export interface ImageChangedEvent {
+  changeEvent: ImageChangeType;
+}
+
+@Component({
+  selector: 'mdm-theme-image',
+  templateUrl: './theme-image.component.html',
+  styleUrls: ['./theme-image.component.scss']
+})
+export class ThemeImageComponent implements OnInit {
+  @Input() apiPropertyId: Uuid;
+  @Output() imageSavedEvent = new EventEmitter();
+  @Output() imageChangedEvent = new EventEmitter<ImageChangedEvent>();
+
+  currentUser: any;
```

Comment:
According to the constructor when `securityHandler.getCurrentUser()` is used, this should be `currentUser: UserDetails`

Reply:
Added correct type.